<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pic_show</class>
 <widget class="QDialog" name="pic_show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="pic_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>82</width>
     <height>156</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="file_Btn">
      <property name="text">
       <string>本地文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pre_picBtn">
      <property name="text">
       <string>上一张</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="next_picBtn">
      <property name="text">
       <string>下一张</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_Btn">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_Btn">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
